client/app/bundles/course/users/components/UserProfileCard.tsx

```diff
@@ -0,0 +1,193 @@
+import PropTypes from 'prop-types';
+import { FC } from 'react';
+import {
+  Avatar,
+  Box,
+  Card,
+  CardContent,
+  Grid,
+  Link,
+  Paper,
+  Typography,
+} from '@mui/material';
+import { scroller } from 'react-scroll';
+import { blue, green, orange } from '@mui/material/colors';
+
+interface Props {
+  name: string;
+  imageUrl: string;
+  email: string;
+  role: string;
+  exp?: number;
+  level?: number;
+  experiencePointsRecordsUrl?: string;
+  manageEmailSubscriptionUrl?: string;
+  achievementCount?: number;
+}
+
+const UserProfileCard: FC<Props> = ({
+  name,
+  imageUrl,
+  email,
+  role,
+  exp,
+  level,
+  experiencePointsRecordsUrl,
+  manageEmailSubscriptionUrl,
+  achievementCount,
+}: Props) => {
+  const styles = {
+    courseUserImage: {
+      height: 140,
+      width: 140,
+    },
+    statsContainer: {
+      '& a': {
+        textDecoration: 'none',
+      },
+      '& .user-stats-card': {
+        margin: '4px 4px 4px 0px',
+        padding: '12px',
+        minWidth: '100px',
+        maxWidth: '144px',
+        transition: 'background-color 0.5s, color 0.5s',
+      },
+      '& :nth-child(2n):hover .user-stats-card,& .user-stats-card:nth-child(2n):hover':
```

Comment:
Ive got this error for using nth-child.

![image](https://user-images.githubusercontent.com/42570513/171787914-b943dfc8-f587-434d-abed-ef43480fa272.png)

Reply:
Shifted to course/users.scss since we need nth-child for this particular styling. A bit overkill to use emotion's `emotion-disable-server-rendering-unsafe-selector-warning-please-do-not-use-this-the-warning-exists-for-a-reason` flag, even though we are not doing server-side rendering.